app/views/schools/dashboards/_update_your_school_profile.html.erb

```diff
@@ -8,13 +8,20 @@
     <p class="govuk-notification-banner__heading">
       <p>
         Before you can receive requests, you need to complete your school profile.
+
         <span class="govuk-heading-s govuk-!-margin-top-4">
           <%= link_to 'Complete your school profile',
             Feature.enabled?(:task_progress_on_boarding) ? schools_on_boarding_progress_path : schools_on_boarding_profile_path,
             class: 'govuk-link'
           %>
         </span>
       </p>
+      <p class = "govuk-body-s">
+        <small>
```

Comment:
Question for @GayeCord as to whether this should be in small text?

My view is probably not - on our GIT funnels, PN references are in the same font size as other body text

Reply:
Fair point - I drew inspiration from the GIT homepage, which uses the `small` tag, as using full size ended up looking oversized (in my view)

**GIT Homepage Email Funnel**
![image](https://github.com/DFE-Digital/schools-experience/assets/35870975/b4456619-c107-48c9-891e-558ba824a09a)

**Without `<small>` text**

![image](https://github.com/DFE-Digital/schools-experience/assets/35870975/8727bd4b-1931-43b2-9562-e644bdfcef64)